infrastructure/environments/dplplat01/configuration/mariadb-servers/templates/mariadb-server.template.yaml

```diff
@@ -46,8 +46,8 @@ spec:
     default_storage_engine=InnoDB
     innodb_buffer_pool_size=3200MB
     innodb_autoinc_lock_mode=2
-    max_allowed_packet=256M
     binlog_format=row
+    max_allowed_packet=1GB
```

Comment:
What was the issue?

Reply:
a production recommendation by the people behind MariaDB